features/works/work_browse.feature

```diff
@@ -178,3 +178,9 @@ Scenario: Can also browse work indexed by language
       Then I should see "1 Work in Deutsch"
     When I browse works in language "Persian"
       Then I should see "0 Works in Persian"
+
+Scenario: Work blurb includes an HTML comment containing the unix epoch of the updated time
+  Given it is currently 2025-04-12 17:00 UTC
```

Comment:
You can freeze the time instead (`time is frozen at  ...`) so that you don't have to use `be_within` for the number. That way we can be sure that the test won't end up flaky if it ever ends up taking longer than 5 seconds for some reason